packages/compass-indexes/src/components/indexes-table/badge-with-icon-link.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  css,
+  Badge,
+  BadgeVariant,
+  Icon,
+  Link,
+  uiColors,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+const badgeStyles = css({
+  gap: spacing[2],
+});
+
+const linkStyles = css({
```

Comment:
How do you feel about adding focus colors to this link? When someone tab navigates to the button it would show the blue ring and the user will then know that hitting enter will open the link. Currently it doesn't have a focus style.